### PR TITLE
Firefox subform drag fix

### PR DESCRIPTION
**Pull request by JackKellyUK**
Pull Request for Issue #37974.

### Summary of Changes

Issue looks to be caused by a bug with Firefox

- https://stackoverflow.com/questions/15967786/unable-to-drag-a-button-element-in-firefox
- https://bugzilla.mozilla.org/show_bug.cgi?id=568313

Reworked subform drag button styling, increasing span click area to circumvent browser bug.

### Testing Instructions

Test in multiple browsers **including** Firefox
1. Open the 'System - HTTP Headers' plugin
2. Add at least 2 subform fields to 'Force HTTP Headers'
3. Check drag button functions correctly

### Actual result BEFORE applying this Pull Request

Only part of the subform drag button functioned in Firefox

### Expected result AFTER applying this Pull Request

Drag button is fully clickable in all browsers

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: build/media_source/templates/administrator/atum/scss/blocks/_form.scss [hunk 1]
```diff
@@ -176,8 +176,12 @@ div.subform-repeatable-group {
         top: 50%;
         right: 100%;
         margin-top: -27px;
-        line-height: 52px;
         border-radius: $border-radius 0 0 $border-radius;
+        padding: 0;
+
+        span {
+          padding: 1.5rem 0.5rem;
+        }
```
Comment (richard67) [inline build/media_source/templates/administrator/atum/scss/blocks/_form.scss]:
```suggestion
        padding: 0;
        border-radius: $border-radius 0 0 $border-radius;

        span {
          padding: 1.5rem .5rem;
        }
```
Fix SCSS code style errors reported by Drone here: https://ci.joomla.org/joomla/joomla-cms/60559/1/25

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f6021cf1b6e19e554b3da0b1000bbc6f0e9f162f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39576">issues.joomla.org/tracker/joomla-cms/39576</a>.</sub>

**Comment by HLeithner** [comment]
@dgrammatiko can you have a look at this?

**Comment by dgrammatiko** [comment]
I have tested this item :white_check_mark: successfully on f6021cf1b6e19e554b3da0b1000bbc6f0e9f162f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39576">issues.joomla.org/tracker/joomla-cms/39576</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39576">issues.joomla.org/tracker/joomla-cms/39576</a>.</sub>

**Comment by roland-d** [comment]
Thank you

**Pull request merged**